charts/csi-isilon/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: csi-isilon
-version: 2.11.0
-appVersion: "2.11.0"
+version: 2.11.1
+appVersion: "2.11.1"
 kubeVersion: ">= 1.21.0"
```

Comment:
Someone from CSM could correct me, but I don't think you want to increment version here since you're merging to release-v1.11.0.

Reply:
Confirmed with Shayna and removed the version increment.